go.mod

```diff
@@ -3,13 +3,13 @@ module github.com/ONSdigital/dp-observation-api
 go 1.13
```

Comment:
A reminder for next time to look. at upgrading go version to 1.15 and update ci build and unit to version 1.15.2 (or latest)